src/rachanaGJun20/Array/ArmstrongNumber.java

```diff
@@ -0,0 +1,35 @@
+package rachanaGJun20.Array;
+
+import java.util.Scanner;
+
+public class ArmstrongNumber {
+
+	//method will find whether given number is Armstrong 
+	boolean isArmstrongNumber(int number) {
+		int remainder = 0;
+		int sum = 0;
+		int givenNumber = number;
+
+		while (number != 0) {
+			remainder = number % 10;
+			number = number / 10;
+			sum = sum + (remainder * remainder * remainder);
```

Comment:
Suggestion: you can use  Math.pow function  as well
No of Variables can be reduced.

Reply:
Sure will Update.. Thank you :)